hatch/src/assets/tests/generator.rs

```diff
@@ -8,16 +8,22 @@ use std::path::PathBuf;
 fn generate_one_without_directories() {
   let test_asset = ProjectAsset::new(PathBuf::from("./"), String::from("test.test"), String::from("test"));
 
-  generate_one(&test_asset);
+  if let Err(e) = generate_one(&test_asset) {
+    panic!(e);
```

Comment:
Im not super happy with the use of panic!

Is there a better way to fail here?

Reply:
This is how Rust tests work, they detect failure via panic.

Reply:
Ok